polaris_bitgo/bitgo/integration.py

```diff
@@ -154,24 +156,29 @@ def submit_deposit_transaction(
         if not response:
             raise BitGoAPIError("Error in BitGo send transaction")
 
-        return self._poll_stellar_transaction_information(txid=response["txid"])
+        return self._poll_stellar_transaction_information(
+            transaction_id=response["id"], bitgo=bitgo
+        )
 
-    def _poll_stellar_transaction_information(self, txid: str) -> dict:
+    def _poll_stellar_transaction_information(
+        self, transaction_id: str, bitgo: BitGo
```

Comment:
Since `transaction_id` and `bitgo` are only used for getting `stellar_transaction_id`, wouldn't it make more sense to have just `stellar_transaction_id` as parameter?